github-pullrequest-plugin/src/test/resources/org/jenkinsci/plugins/github_integration/its/WorkflowITest/workflow-definition2.groovy

```diff
@@ -0,0 +1,31 @@
+package org.jenkinsci.plugins.github_integration.its.WorkflowITest
+
+
+try {
+    node('master') {
+        step([
+                $class       : 'GitHubPRStatusBuilder',
+                statusMessage: [
+                        content: "Run #${env.BUILD_NUMBER} started"
+                ]
+        ])
+        setGitHubPullRequestStatus state: 'PENDING', context: 'custom-context1', message: "Run #${env.BUILD_NUMBER} started"
+        setGitHubPullRequestStatus state: 'PENDING', context: 'custom-context2', message: "Run #${env.BUILD_NUMBER} started"
+
+        sh 'sleep 10 && env'
+
+        setGitHubPullRequestStatus state: 'SUCCESS', context: 'custom-context1', message: 'Tests passed'
+        setGitHubPullRequestStatus state: 'SUCCESS', context: 'custom-context2', message: 'Tests passed'
+        step([
```

Comment:
The error I get when I run this test is
```
java.lang.IllegalStateException: Can't get remote GH repo for it-job
```
I think this error comes when the "global" jenkins settings do not configure a GitHub server that matches the GitHub repository associated with the project. I don't know how that translates to the environment the IT test creates.